src/main/java/com/maxmind/minfraud/request/Email.java

```diff
@@ -21,22 +22,217 @@ public final class Email extends AbstractModel {
     private final boolean hashAddress;
     private final String domain;
     private static final Map<String, String> typoDomains;
+    private static final Map<String, String> equivalentDomains;
+    private static final Map<String, Boolean> fastmailDomains;
+    private static final Map<String, Boolean> yahooDomains;
 
     static {
-        HashMap<String, String> m = new HashMap<>() {{
+        HashMap<String, String> typoDomainsMap = new HashMap<>() {{
             // gmail.com
-            put("35gmai.com", "gmail.com");
-            put("636gmail.com", "gmail.com");
+            put("gmai.com", "gmail.com");
             put("gamil.com", "gmail.com");
-            put("gmail.comu", "gmail.com");
+            put("gmali.com", "gmail.com");
             put("gmial.com", "gmail.com");
             put("gmil.com", "gmail.com");
-            put("yahoogmail.com", "gmail.com");
+            put("gmaill.com", "gmail.com");
+            put("gmailm.com", "gmail.com");
+                put("gmailo.com", "gmail.com");
+                put("gmailyhoo.com", "gmail.com");
+                put("yahoogmail.com", "gmail.com");
             // outlook.com
-            put("putlook.com", "outlook.com");
-        }};
+                put("putlook.com", "outlook.com");
+            }};
+        typoDomains = Collections.unmodifiableMap(typoDomainsMap);
 
-        typoDomains = Collections.unmodifiableMap(m);
+        HashMap<String, String> equivalentDomainsMap = new HashMap<>() {{
+                put("googlemail.com", "gmail.com");
```

Comment:
Hmm the indentation of the previous HashMap block aligns these `put(...)` with the `M` in `Map`, but these alight with the `S` in `String